pysnooper/tracer.py

```diff
@@ -117,6 +119,8 @@ def __init__(self, target_code_object, write, truncate, watch=(),
         self.prefix = prefix
         self.overwrite = overwrite
         self._did_overwrite = False
+        self.threads_info = threads_info
+        self.padding = 0
```

Comment:
I think `padding` and `set_padding` sound too generic. They should indicate that they specifically apply to threads.

Reply:
See me next comment on padding on everything by default to keep the alignment between snooped with threads info turned on and turned off in the same output.

Reply:
I'm just talking about the names. I would go for `thread_info_padding` and `set_thread_info_padding`